nemo/core/neural_modules.py

```diff
@@ -322,48 +399,84 @@ def _validate_config_file(cls, config_file, section_name=None):
         return loaded_config
 
     @classmethod
-    def import_from_config(cls, config_file, section_name=None, overwrite_params={}):
+    def deserialize(cls, configuration, name=None, overwrite_params={}):
```

Comment:
how is this different from import_from_config?

Reply:
Two different "interfaces":
 * import_from_config - accepts config name
 * deserialize - accept configuration dictionary (that may or may not come from a loaded configuration file)

This modularization was necessary for (analogical) import_from_config/deserialization of neural graphs.